lib/liveValidation/operationValidator.ts

```diff
@@ -327,6 +329,22 @@ export const schemaValidateIssueToLiveValidationIssue = (
           }
           return "";
         }
+      } else if (
+        issue.code === "INVALID_FORMAT" &&
+        isArmCall === false &&
+        issue.message.indexOf("Object didn't pass validation for format arm-id") > -1
```

Comment:
nit: you could replace this with 
`issue.message.includes("Object didn't pass validation for format arm-id")`